tests/kernel/common/src/main.c

```diff
@@ -21,6 +22,20 @@ extern void rand32_test(void);
 extern void timeout_order_test(void);
 extern void clock_test(void);
 
+
+static void test_version(void)
+{
+	u32_t version = sys_kernel_version_get();
+
+	zassert_true(SYS_KERNEL_VER_MAJOR(version) == KERNEL_VERSION_MAJOR,
```

Comment:
I'd send a patch in the same fixing this macro definition so that `val` is properly parenthesized.  It's very unlikely it's going to be called with anything other than a simple expression (such as referencing a variable name or a function call), but you never know.

Reply:
are you referring to the zassert_true macro?

Reply:
No, I'm referring to `SYS_KERNEL_VER_MAJOR()` in `kernel_version.h`

Reply:
> I'd send a patch in the same fixing this macro definition so that val is properly parenthesized.

done